api/routes/reviews.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const router = express.Router();
+const Review = require('../models/review');
+
+// Getting all reviews
+router.get('/', async (req, res) => {
+  try {
+    const reviews = await Review.find();
+    res.json(reviews);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+// create a review
+router.post('/', async (req, res) => {
+  const review = new Review({
+    title: req.body.title,
+    rating: req.body.rating,
+    desc: req.body.desc,
+    lastEdited: req.body.lastEdited,
```

Comment:
last edited can just be a new date (same as created) on the api instead of sent by the client